src/components/AdressSearch/style.ts

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled';
+
+export const SearchBox = styled.div`
+  position: relative;
+`;
+
+export const SearchResultsContainer = styled.div`
+  position: absolute;
+  top: 79.5px;
+  width: 100%;
+  border: 1px solid #eff0f2;
+  border-radius: 0.75rem;
+  padding: 28px 24px 0 24px;
```

Comment:
```suggestion
  padding: 28px 24px 0;
```
`padding: 28px 24px 0;`으로 바꿔서 쓸 수 있어요